libs/cardano-ledger-core/cardano-ledger-core.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-core
-version:            1.14.1.0
+version:            1.14.2.0
```

Addition of a new type class method is a breaking change according to PVP.
```suggestion
version:            1.15.0.0
```

---

I have a general question - I looked at chap and the latest released version of cardano-ledger-core I saw there is 1.13.2.0 and so that is why I stayed with 1.14 but bumped the minor version just to indicate a change. Presumably at the next release, the PVP rules would be satisfied since there was a major version bump that somebody else applied due to his/her braking changes, and which I leveraged for my patch. After releasing, it won't be immediately known why 1.13 was bumped to 1.14 (ie. mine or others' changes), but it won't matter because the major version was bumped in the aggregate.